site/app/templates/submission/homework/SubmitBox.twig

```diff
@@ -374,6 +378,26 @@
                             addLabel('{{ file.name }}', '{{ file.size }}', {{ file.part }}, true);
                             readPrevious('{{ file.name }}', {{ file.part }});
                         {% endfor %}
+
+                        //TODO: move logic for loading submission mode from session storage into module file
+                        const selected_radio_id = document.querySelector('input[name="submission-type"]:checked').id;
+                        if(selected_radio_id !== 'radio-normal'){
+                            for(let idx = 1; idx <= window.num_submission_boxes; idx++){
+                                window.deleteFiles(idx);
+                            }
+
+                            const warning_banner = document.getElementById('submission-mode-warning');
+                            if (!warning_banner.hasChildNodes()){
+                                warning_banner.appendChild( document.createElement('h2') );
+                                warning_banner.firstChild.classList.add('warning');
+                            }
+
+                            message = selected_radio_id === 'radio-bulk' ? 'Warning: Submitting files for bulk upload!' :
+                                'Warning: Submitting files for a student!';
```

Comment:
The message "Warning" carries a lot of context with it. In this case I don't really feel like its much of a negative. Maybe something like "Notice", "Note", or "Attention" could make it feel like the instructor isn't doing anything wrong.

Reply:
Warning here is fine. This is an action with a decent amount of concern (e.g. you're submitting for a student, did you mean to, is it the right student, etc.) versus a regular upload.

Reply:
I agree, a warning is the right phrasing for submission for a student to make the instructor make sure they don't accidentally wreck someone's submission history like by uploading to the wrong person or accidentally charging a student a late day.

For bulk upload its less of a concern so I can see that saying something like "You are uploading in bulk upload mode" instead of saying warning